src/embind/embind_gen.js

```diff
@@ -373,6 +379,9 @@ var LibraryEmbind = {
         def.printModuleEntry(this.typeToJsName.bind(this), out);
       }
       out.push('}');
+      if (this.usedEmbindString) {
+        out.push('type EmbindString = ArrayBuffer|Uint8Array|Uint8ClampedArray|Int8Array|string;\n');
```

Comment:
Should we avoid outputting this more than once?

Reply:
This will only happen once per TS generation.